### PR TITLE
Implement account.sendParcel

### PR DESCRIPTION
**Pull request by sgkim126**

### CODE REVIEW

File: src/rpc/account.ts [hunk 1]
```diff
@@ -169,6 +172,39 @@ export class AccountRpc {
         });
     }
 
+    public sendParcel(params: {
```
Comment (joojis) [inline src/rpc/account.ts]: Could you add typedoc comments?

File: src/core/Parcel.ts [hunk 2]
```diff
@@ -136,14 +136,17 @@ export class Parcel {
 
     public toJSON() {
         const { nonce, fee, networkId, action } = this;
-        if (!nonce || !fee) {
+        if (!fee) {
             throw Error("Nonce and fee in the parcel must be present");
```
Comment (joojis) [inline src/core/Parcel.ts]: It seems the error message should be changed also

File: src/rpc/account.ts [hunk 3]
```diff
@@ -169,6 +172,39 @@ export class AccountRpc {
...
+        parcel: Parcel;
+        account: PlatformAddress | string;
+        passphrase?: string;
+    }): Promise<{ hash: H256; nonce: U256 }> {
+        const { parcel, account, passphrase } = params;
+        if (!PlatformAddress.check(account)) {
+            throw Error(
+                `Expected the second argument to be a PlatformAddress value but found ${account}`
```
Comment (joojis) [inline src/rpc/account.ts]: the second argument -> account parameter, since it isn't a positional argument.

File: src/rpc/account.ts (hunk 1 above)
Comment (sgkim126) [reply src/rpc/account.ts]: done

File: src/core/Parcel.ts (hunk 2 above)
Comment (sgkim126) [reply src/core/Parcel.ts]: done

File: src/rpc/account.ts (hunk 3 above)
Comment (sgkim126) [reply src/rpc/account.ts]: done

**Comment by sgkim126** [comment]
Travis fails because account_sendParcel is not applied to the docker image.
I'll re-run it after changing the docker image.

**Comment by sgkim126** [comment]
@joojis please review it

**Pull request merged**